test/test_cloud_image_validator.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+
+from main.cloud_image_validator import CloudImageValidator
+from cloud.terraform.terraform_configurator import TerraformConfigurator
+from cloud.terraform.terraform_controller import TerraformController
+from result.reporter import Reporter
+from test_suite.suite_runner import SuiteRunner
+
+
+class TestCloudImageValidator:
+    test_resources_file = '/fake/test/resources_file.json'
+    test_output_file = '/fake/test/output_file.xml'
+    test_parallel = True
+    test_debug = True
+    test_instances = ['test instance 1', 'test instance 2']
```

Comment:
I think I'm being very picky, but it triggers my OCD to see space separated names instead of underscore separation :S